### PR TITLE
Feature/more entities

### PR DESCRIPTION
**Pull request by JakobRathberger**

### CODE REVIEW

File: .github/workflows/deploy-valuesimulator.yaml [hunk 1]
```diff
@@ -48,10 +52,16 @@ jobs:
           ssh-private-key: ${{ secrets.DEPLOYMENT_VALUESIM_PRIVKEY }}
       - name: create .ssh/known_hosts
         run: ssh-keyscan -H -t rsa -v vm09.htl-leonding.ac.at  >> ~/.ssh/known_hosts
```
Comment (quirinecker) [inline .github/workflows/deploy-valuesimulator.yaml:54]: maybe abstract the url in a github secret

File: .github/workflows/deploy-valuesimulator.yaml [hunk 2]
```diff
@@ -48,10 +52,16 @@ jobs:
...
+      - name: Copy required files
+        run: |
+          scp -r docker-compose.yml iotadmin@vm09.htl-leonding.ac.at:/opt/docker/leo-iot
       - name: Run Docker Container
         run: |
           ssh iotadmin@vm09.htl-leonding.ac.at "
```
Comment (quirinecker) [inline .github/workflows/deploy-valuesimulator.yaml:60]: maybe abstract the url in a github secret

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+FROM ubuntu:impish-20211102 as run
+
+WORKDIR /app
+
+COPY --from=build /app/target/*-runner /app/application
+
+EXPOSE 8080
+
+CMD ["./application"]
```
Comment (quirinecker) [inline Dockerfile:30]: i dont know if the service allready runs with a databse. There can be problems on startup when the application startup time ist faster then the database startup time. You may need a little wait script. Please take a look at that:

https://github.com/quirinecker/leocloud/tree/master/appsrv/docker/scripts

Comment (quirinecker) [reply Dockerfile:30]: note this is the original repository:

https://github.com/caberger/leocloud/tree/master/appsrv/docker/scripts

**Review by quirinecker — COMMENTED** [review]
looks very good i was not focusing to much on the code because it was much code but i wrote a few comments for configuration stuff.

One suggestion for your future java code ist lombok. Java code is fine without it but it makes things a little easier for you..

https://projectlombok.org/

When you have time and motiviation you can look into it if you want.

File: Dockerfile (hunk 3 above)
Comment (JakobRathberger) [reply Dockerfile:30]: fixed this by adding a restart on failure in the docker-compose.yml file

Comment (quirinecker) [reply Dockerfile:30]: Ohh i see did not see that

File: .github/workflows/deploy-valuesimulator.yaml (hunk 1 above)
Comment (JakobRathberger) [reply .github/workflows/deploy-valuesimulator.yaml:54]: https://vm81.htl-leonding.ac.at/issue/leoiot-122
will be done in the future

File: .github/workflows/deploy-valuesimulator.yaml (hunk 2 above)
Comment (JakobRathberger) [reply .github/workflows/deploy-valuesimulator.yaml:60]: https://vm81.htl-leonding.ac.at/issue/leoiot-122
will be done in the future

**Pull request merged**